src/components/navigation/Navigation.css

```diff
@@ -0,0 +1,28 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+.default-link {
+    border-bottom: 1px solid transparent;
+}
+
+.active-link {
+    border-bottom: 1px solid black;
```

Comment:
misschien hier de tekst nog een andere kleur geven zodat je nog duidelijker ziet welke active is. En ik zou de standaard styling voor a href weghalen, want daardoor valt je streepje nu niet goed op.